webapp/pages/profile/_id/_slug.vue

```diff
@@ -266,6 +267,35 @@ export default {
       voted: false,
       page: 1,
       pageSize: 6,
+      modalsData: {
+        delete: {
```

Comment:
If find this slightly inconsistently named - down below it's called `disableCallback` and `reportCallback` and here it's just `delete`.

Reply:
This was related to my discussion for the right path with @mattwr18 and has nothing to do with real programming! @roschaefer